### DISCUSSION TITLE
✅ Change Search Form Icon?

### DISCUSSION THREAD

**Discussion by nkoerner93** [elementor/elementor#14583] [Feature Request]
Hey Elementor Commmunity,

We have custom SVG's that we use for our icons, is it possible to change the search icon in the elementor search form with an SVG? Would be great if someone could help me out here.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.23! 🥳 

You can now use the [new Search widget with Live Results](https://elementor.com/help/search-widget/), and select your own icons, it also includes the option to choose your query as well!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
